chemprop/train/run_training.py

```diff
@@ -200,7 +200,12 @@ def run_training(args: TrainArgs,
         else:
             debug(f'Building model {model_idx}')
             model = MoleculeModel(args)
-
+            
+        # Optionally, overwrite weights:
+        if args.frzn_mpnn | (args.frzn_ffn_layers > 0):
+            debug(f'Loading mpn parameters from {args.checkpoint_frzn}')
+            model = load_frzn_model(model=model,path=args.checkpoint_frzn, current_args=args, logger=logger)     
+        
         debug(model)
         debug(f'Number of parameters = {param_count(model):,}')
```

Comment:
Does this parameter count update acknowledge frozen parameters? I wonder if there's a way we can provide a total parameter count and a frozen parameter count. Not a requirement but would be nice.

Reply:
Added this!